src/file.ts

```diff
@@ -3707,6 +3714,8 @@ class File extends ServiceObject<File, FileMetadata> {
    * @param {string} [userProject] The ID of the project which will be
    *     billed for the request.
    * @param {number} [generation] If present, selects a specific revision of this object.
+   * @param {string} [restoreToken] If this is a soft-deleted object in an HNS-enabled bucket, returns the restore token which will
```

Comment:
nit: change description to be precise

Returns an option that must be specified when getting a soft-deleted object from an HNS-enabled bucket that has a name/generation conflict with another object in the same bucket.